simtbx/kokkos/SConscript

```diff
@@ -204,10 +204,14 @@ print('-'*79)
 kokkos_cxxflags = subprocess.check_output(
   ['make', '-f', 'Makefile.kokkos', 'print-cxx-flags'],
   cwd=os.environ['KOKKOS_PATH'])
-kokkos_cxxflags = kokkos_cxxflags.split(b'\n')
-kokkos_cxxflags = kokkos_cxxflags[1].decode('utf8').split()
-if kokkos_cxxflags and kokkos_cxxflags[0] == 'echo':
-  kokkos_cxxflags = [f.strip('"') for f in kokkos_cxxflags[1:]]
+#print('5'*79)
+#print(os.environ['KOKKOS_CXXFLAGS'])
+#print('5'*79)
+#kokkos_cxxflags = kokkos_cxxflags.split(b'\n')
+#kokkos_cxxflags = kokkos_cxxflags[1].decode('utf8').split()
+kokkos_cxxflags = os.environ['KOKKOS_CXXFLAGS'].split()
```

Comment:
@Trzs I think this part removes the "echo" that is showing up. I remember we saw this earlier and this is probably why it's here.